screeps-game-api/src/constants.rs

```diff
@@ -448,8 +458,16 @@ pub mod look {
     }
 }
 
+/// Creep part types.
+///
+/// *Note:* This constant's `TryFrom<Value>`, `Serialize` and `Deserialize`
+/// implementations only operate on made-up integer constants. If you're ever
+/// using these impls manually, you need to use the `__part_num_to_str` and
+/// `__part_str_to_num` JavaScript functions to convert.
```

Comment:
hopefully you can copy paste the previous example...
Or maybe make one at the module level and refer to it here?

Reply:
Module-level doc sounds like a good idea.